server/services/health-service.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs')
+
+async function writeLiveness () {
+  fs.writeFile('/tmp/calculation-service-healthz.txt', Date.now(), (err) => {
```

Comment:
The liveness probe doesn't actually do any checking. Should it check that the message queue server is still connected?

Reply:
The role of a liveness probe is to monitor whether the service is still alive.  If it monitored the dependencies such as queues and database then a lost dependency would put the app in a crash loop which wouldn't be the intended behaviour.

Readiness probs are used to check dependencies, if one is unavailable traffic is not routed to that service.  This service doesn't receive any traffic so a readiness probe is redundant.